recipes/homes.rb

```diff
@@ -24,7 +24,7 @@
 end
 
 nfs_export '/data/homes/' do
-  network (1..26).map { |octet| '10.1.1.' + octet.to_s + '/32' }
+  network (1..254).map { |octet| '10.162.136.' + octet.to_s + '/24' }
```

Comment:
Yeah, but I think @armiller is right. This creates exports for 10.162.136.1/24, 10.162.136.2/24 ... 10.162.136.254/24
So I'm guessing it should either be /32, or just a single 10.162.136.0/24. Right?